pkg/validation/policy_template.go

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright 2025 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	"encoding/json"
+	"fmt"
+
+	kyvernoapiv1 "github.com/kyverno/kyverno/api/kyverno/v1"
+
+	kubermaticv1 "k8c.io/kubermatic/sdk/v2/apis/kubermatic/v1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+var (
+	validVisibilities = sets.New(kubermaticv1.PolicyTemplateVisibilityGlobal, kubermaticv1.PolicyTemplateVisibilityProject, kubermaticv1.PolicyTemplateVisibilityCluster)
```

Comment:
```suggestion
	validPolicyVisibilities = sets.New(kubermaticv1.PolicyTemplateVisibilityGlobal, kubermaticv1.PolicyTemplateVisibilityProject, kubermaticv1.PolicyTemplateVisibilityCluster)
```

this variable in shared across whole `validation` package, need to have a bit more specific name.